warre/inventory/inventory.html

```diff
@@ -23,16 +23,110 @@
 			crossorigin="anonymous"></script>
 
 	<!-- internal resource links -->
+	<!--
 	<script src="../js/global_on_page_load.js"></script>
 	<script src="../js/banner.js"></script>
 	<script src="../js/authentication.js"></script>
+	-->
+	<script src="js.cookie.js"></script>
 	<link href="inventory.css" type="text/css" rel="stylesheet">
 
 	<!-- initialize page -->
 	<script>
 		jQuery(document).ready(function(){
-			on_page_ready();
+
+		$("id-input").focus();
+
+		function selectCI() {
+			ci = true;
+			co = false;
+			stat = false;
+			if ($('#chk-out-li').hasClass('active')) {
```

Comment:
It seems like you're doing this and the inverse a lot, can you abstract it to a function to be cleaner?